JFrogVSExtension.sln

```diff
@@ -5,6 +5,8 @@ VisualStudioVersion = 17.1.32407.343
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "JFrogVSExtension", "JFrogVSExtension\JFrogVSExtension.csproj", "{6443B797-2478-4A1D-BECA-28E24C1F1F41}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UnitTestJfrogVSExtension", "UnitTestJfrogVSExtension\UnitTestJfrogVSExtension.csproj", "{DB770B26-1629-45D0-8526-8902F83731B2}"
```

Comment:
Please check if can be a CONST somewhere.
Also - see how can we separate the tests from the solution

Reply:
It is possible to define in the build command which project we want to build so its ok